### DISCUSSION TITLE
What does ssslv3 alert ahdshake error mean in API?

### DISCUSSION THREAD

**Discussion by RatulSaha** [supabase/supabase#7545] [Questions]
When calling the API, we are quite often getting the following message.

```{"message":"FetchError: request to https://xxxx.supabase.co/rest/v1/xxx?select=key%2Cbody&key=eq.xxx failed, reason: write EPROTO 140349404755904:error:14094410:SSL routines:ssl3_read_bytes:sslv3 alert handshake failure:../deps/openssl/openssl/ssl/record/rec_layer_s3.c:1545:SSL alert number 40\n","details":"","hint":"","code":"EPROTO"}```

What does this mean? Are we calling far too frequently than the Supabase instance can handle? We are calling this from a SQS queue consumer, so it's not unbounded parallel.

**Comment by ihadabs** [comment]
Same here, any update on this?

**Comment by pierrenel** [comment]
I see this when using a custom API domain (which I'm paying for) - if I instead use https://{project_id}.supabase.co it works 😬
